changelogs/unreleased/3803-disallow-multiline-strings-with-single-quotes.yml

```
@@ -0,0 +1,6 @@
+description: Trying to write a single quoted string over multiple lines now raises a syntax error.
+change-type: major
+destination-branches: [master, iso5]
+sections: {
+  bugfix: "{{description}}"
```

```suggestion
  bugfix: "{{description}}"
  deprecation-note: "{{description}} This was previously allowed due to a bug."
```
I think we want this in both sections.